resources/lib/playerinfo.py

```diff
@@ -72,8 +82,12 @@ def onAVStarted(self):  # pylint: disable=invalid-name
         self.quit.clear()
         self.update_position()
         self.update_total()
+        self.push_upnext()
 
-        Thread(target=self.stream_position, name='StreamPosition').start()
+        # Only start StreamPosition Thread when necessery
```

Comment:
Small typo.
```suggestion
        # Only start StreamPosition Thread when necessary
```

Also could you elaborate (in comments) when or why this is/isnot  necessary ?

Reply:
Also, I think the variable name `streampos` is not clear. I assumed this was holding a position, like last_pos.
Not sure about a better name, maybe "positionthread"?

Reply:
StreamPosition thread keeps running when watching multiple episode with "Up Next", only start StreamPosition thread when it doesn't exist yet.